usvm-ts/src/main/kotlin/org/usvm/machine/state/TsState.kt

```diff
@@ -64,6 +64,7 @@ class TsState(
     var associatedFunction: UPersistentHashMap<UConcreteHeapRef, TsFunction> = persistentHashMapOf(),
     var closureObject: UPersistentHashMap<String, UConcreteHeapRef> = persistentHashMapOf(),
     var boundThis: UPersistentHashMap<UConcreteHeapRef, UHeapRef> = persistentHashMapOf(),
+    val previousState: TsState? = null,
```

Comment:
What is it? Why? Probably, if you need it, something goes wrong. State are designed to be independent

Reply:
For debugging only.